test_js/float.test.ts

```
@@ -0,0 +1,70 @@
+		expect(b.gte(a)).toBeTruthy();
+		expect(c.gte(a)).toBeTruthy();
+
+		expect(a.eq(c)).toBeTruthy();
+		expect(zero.isZero()).toBeTruthy();
+
+		expect(a.max(b)?.value!.eq(b)).toBeTruthy();
+		expect(a.min(b)?.value!.eq(a)).toBeTruthy();
+	});
+
+	it('should test math ops', async function () {
+		const a = Float.parse("3.14")?.value!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)